bindings/python/tools/python/ostk/astrodynamics/display.py

```diff
@@ -0,0 +1,102 @@
+################################################################################################################################################################
+
+# @project        Open Space Toolkit ▸ Astrodynamics
+# @file           bindings/python/tools/python/ostk/astrodynamics/display.py
+# @author         Remy Derollez <remy@loftorbital.com>
+# @license        Apache License 2.0
+
+################################################################################################################################################################
+
+import plotly.graph_objs as go
+
+################################################################################################################################################################
```

Comment:
Add functions to overlay orbit, ground positions, accesses, etc on 2D/3D maps?

Reply:
Alright, I'll add those yup

Reply:
So for now, I am keeping the task of creating the `data` object on the user side and if needed I'll revisit later